### PR TITLE
Add modal managed kafka page

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Created a managed kafta modal form
- Created a [script to build](https://intl.transitionelement.com/modal/index.cgi?title=Canonical%E2%80%99s+Managed+Kafka&description=Apache+Kafka%2C+managed+by+the+open+source+experts.&q1=What+version+of+Kafka+are+you+running%3F&q1-answers=1.x%0D%0A2.x&q1-code=version+of+kafka&q1-other=&q2=Where+are+you+running+it%3F&q2-answers=Amazon+Web+Services%0D%0AMicrosoft+Azure%0D%0AGoogle+Cloud%0D%0AKubernetes%0D%0AOpenStack%0D%0AOracle%0D%0AVMWare%0D%0ACisco%0D%0AOther+public+cloud%0D%0AOther+on-premises%0D%0A&q2-code=where-are-you+running+kafka&q2-other=&q3=And+are+you+using+any+other+apps+with+it%3F&q3-answers=ElasticSearch%0D%0AOpen+Source+Mano%0D%0AInfluxDB%0D%0APostgreSQL%0D%0AGraylog%0D%0AGrafana%0D%0AMongoDB%0D%0AMySQL%0D%0APrometheus%0D%0ACassandra&q3-code=using+any+other+apps+with+kafka&q3-other=&q4=&q4-answers=&q4-code=&q4-other=&question=Can+you+summarize+in+a+few+words+your+project+and+what+assistance+you%27re+looking+for%3F&ty-title=Thank+you+for+getting+in+touch+about+managed+Kafka&ty-text=A+member+of+staff+will+get+back+to+you+within+one+working+day.&form-id=&a=build) these forms

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed/kafka#get-in-touch
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1ExHVIv4GkX7AJUueAt7os1ryawAWO7Zj2HymU0o31YA/edit#)
- Fill in the form and see that it submits properly

### CODE REVIEW

File: templates/shared/forms/interactive/managed-kafka.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <hr />
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/managed-kafka.html:5]: Can you remove this `hr` from all interactive forms and the creation script, please.

File: templates/shared/forms/interactive/managed-kafka.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      <h2 class="p-modal__title" id="modal-title">Canonical’s Managed Kafka</h2>
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/managed-kafka.html:6]: Can you add the class `u-sv1` to this header for all interactive forms and the creation script, please.

File: templates/shared/forms/interactive/managed-kafka.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <p id="modal-description" class="u-no-max-width u-no-margin--bottom">Apache Kafka, managed by the open source experts.</p>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">What version of Kafka are you running?</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="version-of-kafka-1x">
+            <label for="version-of-kafka-1x">1.x
```
Comment (anthonydillon) [inline templates/shared/forms/interactive/managed-kafka.html:16]: Can you stop the new line at the end of the label as it's is appearing in the resulting message.

Comment (anthonydillon) [reply templates/shared/forms/interactive/managed-kafka.html:16]: image

Comment (pmahnke) [reply templates/shared/forms/interactive/managed-kafka.html]: done

File: templates/shared/forms/interactive/managed-kafka.html (hunk 2 above)
Comment (pmahnke) [reply templates/shared/forms/interactive/managed-kafka.html]: on the H2... yes

File: templates/shared/forms/interactive/managed-kafka.html (hunk 1 above)
Comment (pmahnke) [reply templates/shared/forms/interactive/managed-kafka.html]: yes - all as in go fix the other ones?

**Pull request merged**